README.md

```diff
@@ -19,92 +21,60 @@ Greetings from the Data Platform Team! We are happy and proud to announce an MVP
-        s3_region_name: "eu-central-1".to_string(), // AWS S3 bucket region
-        start_block_height: 82422587, // the latest block height we've got from explorer.near.org for testnet
-    };
+   // create a NEAR Lake Framework config
+   let config = LakeConfigBuilder::default()
+       .testnet()
+       .start_block_height(82422587)
+       .build()
+       .expect("Failed to build LakeConfig");
 
-    // instantiate the NEAR Lake Framework Stream
-    let stream = near_lake_framework::streamer(config);
+   // instantiate the NEAR Lake Framework Stream
+   let stream = near_lake_framework::streamer(config);
 
-    // read the stream events and pass them to a handler function with
-    // concurrency 1
-    let mut handlers = tokio_stream::wrappers::ReceiverStream::new(stream)
-        .map(|streamer_message| handle_streamer_message(streamer_message))
-        .buffer_unordered(1usize);
+   // read the stream events and pass them to a handler function with
+   // concurrency 1
+   let mut handlers = tokio_stream::wrappers::ReceiverStream::new(stream)
+       .map(|streamer_message| handle_streamer_message(streamer_message))
+       .buffer_unordered(1usize);
 
-    while let Some(_handle_message) = handlers.next().await {}
+   while let Some(_handle_message) = handlers.next().await {}
 
-    Ok(())
+   Ok(())
 }
 
 // The handler function to take the entire `StreamerMessage`
 // and print the block height and number of shards
 async fn handle_streamer_message(
-    streamer_message: near_lake_framework::near_indexer_primitives::StreamerMessage,
+   streamer_message: near_lake_framework::near_indexer_primitives::StreamerMessage,
 ) {
-    eprintln!(
-        "{} / shards {}",
-        streamer_message.block.header.height,
-        streamer_message.shards.len()
-    );
+   eprintln!(
+       "{} / shards {}",
+       streamer_message.block.header.height,
+       streamer_message.shards.len()
+   );
 }
 ```
 
-Video tutorial:
+For more information [refer to the docs](https://docs.rs/near-lake-framework)
+
+### Tutorials
 
-https://youtu.be/GsF7I93K-EQ
+- Video tutorial about [`near-examples/near-lake-accounts-watcher`](https://github.com/near-examples/near-lake-accounts-watcher) https://youtu.be/GsF7I93K-EQ
+- [Migrating to NEAR Lake Framework](https://near-indexers.io/tutorials/lake/migrating-to-near-lake-framework) from [NEAR Indexer Framework](https://near-indexers.io/docs/projects/near-indexer-framework)
 
 ### More examples
 
-- https://github.com/near-examples/near-lake-raw-printer simple example of a data printer built on top of NEAR Lake Framework
-- https://github.com/near-examples/near-lake-accounts-watcher another simple example of the indexer built on top of NEAR Lake Framework for a tutorial purpose
+- [`near-examples/near-lake-raw-printer`](https://github.com/near-examples/near-lake-raw-printer) simple example of a data printer built on top of NEAR Lake Framework
```

Comment:
Do you plan to maintain the printer in a separate repo? I think we should add a reference to the example folder in the printer repo README and archive printer repo

Reply:
I've decided to keep it for now. I'm thinking of removing the `examples` folder from the project or replacing a similar `printer` in `examples` with something different, but haven't come up with what, yet.